.vscode/launch.json

```diff
@@ -10,7 +10,7 @@
       "windows": {
         "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron.cmd"
       },
-      "args" : ["."],
+      "args" : ["./dist/main.js"],
```

Comment:
Would then need to be in some way adapted (e.g. `./source/main/modules/ipc/main.js`)